test/queue.test.js

```diff
@@ -16,6 +16,10 @@ import assert from 'assert';
+};
```

Comment:
We could probably build this string with a `for` loop if you wanted to avoid having a massive value in the file